server/webapp/WEB-INF/rails/webpack/single_page_apps/agents.tsx

```diff
@@ -13,36 +13,57 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import $ from "jquery";
+
+import {SinglePageAppBase} from "helpers/spa_base";
+import {AgentsPage} from "views/pages/agents";
+
+import {AjaxPoller} from "helpers/ajax_poller";
 import m from "mithril";
 import Stream from "mithril/stream";
+// @ts-ignore
```

Comment:
why do we need to do a `ts-ignore`? What the lint is failing for?

Reply:
The following errors pops up if `ts-ignore` is removed: 
`TS7016: Could not find a declaration file for module 'models/agents/agents'. '/Users/kritikasingh/gocd/server/webapp/WEB-INF/rails/webpack/models/agents/agents.js' implicitly has an 'any' type`

Reply:
Some of the files needed does not have an `export`. Hence, `import` is throwing the above mentioned error. For that purpose `ts-ignore` is needed. For the rest, used `require` instead of `import`.